src/CovidAppointmentTable.test.js

```diff
@@ -0,0 +1,73 @@
+import {
+  act,
+  render,
+  screen
+} from '@testing-library/react';
+
+import CovidAppointmentTable from './CovidAppointmentTable';
+
+import prod from '../test/fixtures/api/prod.json';
+import noData from '../test/fixtures/api/no-data.json';
+
+beforeAll(function() {
+  jest.spyOn(window, 'fetch');
+});
+
+describe('the CovidAppointmentTable component', function() {
+  describe('when api data is available', function() {
+    beforeEach(function() {
+      window.fetch.mockResolvedValueOnce({
+        json: async () => prod,
+        ok: true
+      });
+    });
+
+    test('it correctly displays available appointments', async function() {
+      await act(async function() {
+        render(
+          <CovidAppointmentTable />
+        );
+      });
+
+      expect(await screen.findByText(/Berkshire Community College Field House/)).toBeInTheDocument();
+    });
```

Comment:
I think that either this wording should be updated or the expect should be more generic. Making sure that one line of text appears isn't really testing that it displays correctly.

I recommend either:
1. Adding additional `expect`s that give a fuller picture of what we expect to be displayed, or
2. Renaming the test to something like 'it displays the name of a vaccine location' or something like that.